<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchBook</class>
 <widget class="QWidget" name="searchBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统 - 图书查找</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource.qrc">
    <normaloff>:/lib/lib/icon.png</normaloff>:/lib/lib/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="searchTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="searchCountlabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;共找到 0本符合搜索条件的书本&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="bookTable">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>142</number>
       </attribute>
       <column>
        <property name="text">
         <string>ISBN</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>书名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>作者名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>价格</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>出版社</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>返回上一级</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
